From dc5f05eb934ebd285e01febb9a4018512db702b1 Mon Sep 17 00:00:00 2001
From: David Goulet <dgoulet@ev0ke.net>
Date: Sat, 19 Jan 2013 15:53:23 -0500
Subject: [PATCH] Fix: pass opdata when sending message fragment

The inject_message callback was missing the opdata when sending message
fragments.

Signed-off-by: David Goulet <dgoulet@ev0ke.net>
---
 src/message.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/message.c b/src/message.c
index 7c39529..aa71df2 100644
--- a/src/message.c
+++ b/src/message.c
@@ -441,7 +441,7 @@ fragment:
 	/* Fragment and send according to policy */
 	if (!err && messagep && *messagep) {
 	    if (context) {
-		err = fragment_and_send(ops, NULL, context, *messagep,
+		err = fragment_and_send(ops, opdata, context, *messagep,
 			fragPolicy, messagep);
 	    }
 	}
-- 
1.7.10.4

